EarlGrey/Core/GREYElementFinder.m

```diff
@@ -37,8 +37,8 @@ - (NSArray *)elementsMatchedInProvider:(id<GREYProvider>)elementProvider {
   NSParameterAssert(elementProvider);
   I_CHECK_MAIN_THREAD();
   NSMutableArray *matchingElements = [[NSMutableArray alloc] init];
-  for (id element in [elementProvider dataEnumerator]) {
-    @autoreleasepool {
+  @autoreleasepool {
```

Comment:
revert. please do a thorough check first. sync github to internal then internal to external please.